dev/com.ibm.ws.jmx/src/com/ibm/ws/jmx/internal/DelayedMBeanActivator.java

```diff
@@ -1,13 +1,14 @@
-/*******************************************************************************
- * Copyright (c) 2011 IBM Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
+/*
+ * IBM Confidential
```

Comment:
Wrong license header. Also doesn't look like you meant to make any changes here since it's otherwise all formatting. Possible to remove from the PR?

Reply:
Yep, probably cleaner if I just removed my reverted changes to DelayedMBeanActivator, since it shouldn't be touched by any changes